### PR TITLE
Added Heroku deployment button and app.json

### PR DESCRIPTION
**Pull request by torquecoder**
Implemented 1-click heroku installation #21

### CODE REVIEW

**Comment by championswimmer** [comment]
Better naming of PR please.

**Comment by torquecoder** [comment]
Done :)

File: db/models.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const config = require('../config');
 
 
 const db = new Sequelize(
-    secrets.DB.NAME, secrets.DB.USER, secrets.DB.PASSWORD,  {
+    process.env.DATABASE_URL,  {
```
Comment (championswimmer) [inline db/models.js]: This has to be if-else 'ed 
We should  be able to use it both on a VPS (using secrets.json) *as well as* on heroku.

**Comment by championswimmer** [comment]
closing as it is not being updated

**Pull request closed without merging**